packages/fluentui/accessibility/src/behaviors/Grid/gridWithInnerZoneBehavior.ts

```diff
@@ -0,0 +1,23 @@
+import { Accessibility } from '../../types';
```

Comment:
Please use more specific name - something like cardsContainerBehavior. There might be details that would be specific for the use case, let's discuss them offline.